src/uberjob/progress/_console_progress_observer.py

```diff
@@ -81,7 +81,7 @@ def _render(self, state, new_exception_index, exception_tuples, elapsed):
             print_ = partial(print, file=buffer)
             _print_header(print_, elapsed)
             for section in ("stale", "run"):
-                scope_mapping = state.section_scope_mapping.get(section)
+                scope_mapping = state.get(section)
```

Comment:
The three SimpleProgressObserver implementations are now passed the section scope mapping directly so they don't have this awkward expression.